src/components/AdRedirectChannelItem.jsx

```diff
@@ -0,0 +1,33 @@
+import SelectIcon from "@/assets/svgs/icon-select.svg?react";
+import Button from "@/components/ui/Button";
+
+const AdRedirectChannelItem = ({
+  channelId,
+  channelName,
+  thumbnail,
+  isSelected,
+  onSelect,
+}) => {
+  const borderColor = isSelected ? "border-[#5b4dff]" : "border-[#e0dfdf]";
```

Comment:
```suggestion
  const borderColor = isSelected ? "border-indigo-500" : "border-gray-200";
```



피그마에서 제가 작업할 때는 팀원분들이 보기 편하게 실제 색상 기준으로 컬러표를 뽑아둔 것이라, Tailwind 클래스가 포함돼 있지는 않았어요!
그래서 아마 해당 부분에 border-[#5b4dff]처럼 직접 HEX 값을 사용하신 것 같아요~
그렇지만 저희 팀은 전체적으로 Tailwind에서 제공하는 기본 색상 클래스를 쓰고 있고, 현재 다른 컴포넌트들도 모두 일반 Tailwind 색상으로 맞춰져 있어서 통일성 있게 `border-indigo-500` 같은 방식으로 수정하는 건 어떨까요??

Reply:
네 맞습니다!
피그마 기준으로 작업하면서 HEX 값을 그대로 가져와 적용했던 부분인데,
말씀 주신 대로 Tailwind 기본 색상 클래스를 사용하는 게 일관성 면에서 훨씬 좋을 것 같아요.
좋은 피드백 감사합니다 :)